config/findbugs-exclude.xml

```diff
@@ -99,6 +99,10 @@
         <Bug pattern="DE_MIGHT_IGNORE"/>
     </Match>
 
+    <Match>
+        <Class name="com.mongodb.embedded.client.MongoDBCAPIHelper$MongoDBCAPIInitParams"/>
```

Comment:
I've commented in the `MongoDBCAPIInitParams` class as to why this is needed.

Reply:
ack